sandboxed_containers/upgrading-sandboxed-containers.adoc

```diff
@@ -1,27 +1,27 @@
 :_content-type: ASSEMBLY
-[id="upgrade-sandboxed-containers"]
-= Upgrade {sandboxed-containers-first}
+[id="upgrading-sandboxed-containers"]
+= Upgrading {sandboxed-containers-first}
 include::_attributes/common-attributes.adoc[]
-:context: upgrade-sandboxed-containers
+:context: upgrading-sandboxed-containers
 
 toc::[]
 
 You can upgrade the components of {sandboxed-containers-first} by upgrading {sandboxed-containers-operator} and {sandboxed-containers-first} artifacts.
 
 [id="sandboxed-containers-upgrade-operator"]
-== Upgrade {sandboxed-containers-operator}
+== Upgrading {sandboxed-containers-operator}
 
-You can use Operator Lifecycle Manager (OLM) to manually or automatically upgrade the {sandboxed-containers-operator}. You can select manual or automatic upgrade during the initial deployment. In the context of manual upgrades, the web console shows the available updates that can be installed by the cluster administrator.
+You can use Operator Lifecycle Manager (OLM) to manually or automatically upgrade the {sandboxed-containers-operator}. Select manual or automatic upgrade during the initial deployment. For manual upgrades, the web console shows the available updates that can be installed by the cluster administrator.
```

Comment:
@tshwartz You removed "You can" in other areas, I suggest you continue by removing "You can" here just as you have done elsewhere.

Reply:
Done